pkg/chart/loader/load_test.go

```diff
@@ -488,6 +489,118 @@ func TestLoadInvalidArchive(t *testing.T) {
 	}
 }
 
+func TestLoadValues(t *testing.T) {
+	testDatas := []struct {
```

Comment:
nit: `testCases` might be better (and more consistent) naming/terminology here

```suggestion
	testCases := []struct {
```

Reply:
Personally, I prefer:
```
testCases := map[string]struct {
```

Where the map key is the name of the test. It does make the test ordering non-deterministic (which IMHO is a good things). Up to you if you would also prefer a map here.